core/src/main/java/org/kiwix/kiwixmobile/core/help/HelpScreen.kt

```diff
@@ -0,0 +1,205 @@
+    topBar = {
+      Row(
+        modifier = Modifier.fillMaxWidth(),
+        verticalAlignment = Alignment.CenterVertically
+      ) {
+        HelpTopAppBar(navController)
+      }
+    },
+    containerColor = backgroundColor
+  ) { innerPadding ->
+    HelpContent(data, dividerColor, innerPadding)
+  }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun HelpTopAppBar(navController: NavController) {
+  // Retrieve the actionBarSize from the current theme
+  val context = LocalContext.current
+  val actionBarHeight = with(LocalDensity.current) {
+    // Obtain the height defined in the theme (usually 56dp on phones)
+    val styledAttributes =
+      context.theme.obtainStyledAttributes(intArrayOf(android.R.attr.actionBarSize))
+    styledAttributes.getDimension(0, 0f).toDp().also { styledAttributes.recycle() }
+  }
+
+  TopAppBar(
+    modifier = Modifier.height(actionBarHeight), // set the height here
+    title = {
+      Row(
+        modifier = Modifier.fillMaxSize(),
+        verticalAlignment = Alignment.CenterVertically
+      ) {
+        Text(
+          modifier = Modifier.padding(
+            start = dimensionResource(R.dimen.activity_horizontal_margin)
+          ),
+          text = stringResource(id = R.string.menu_help),
+          color = Color.White,
+          fontWeight = FontWeight.SemiBold
+        )
+      }
+    },
+    navigationIcon = {
+      Row(modifier = Modifier.fillMaxHeight(), verticalAlignment = Alignment.CenterVertically) {
+        IconButton(onClick = navController::popBackStack) {
+          Icon(
+            imageVector = Icons.AutoMirrored.Filled.ArrowBack,
+            contentDescription = "Back_Navigation",
+            tint = Color.White
+          )
+        }
+      }
+
+    },
+    colors = TopAppBarDefaults.topAppBarColors(
+      containerColor = Color.Black
+    )
+  )
+}
+
+@Composable
+fun HelpContent(
+  data: List<HelpScreenItemDataClass>,
+  dividerColor: Color,
+  innerPadding: androidx.compose.foundation.layout.PaddingValues
+) {
+  Column(
+    modifier = Modifier
+      .padding(innerPadding)
+  ) {
+    SendReportRow()
+    HelpItemList(data, dividerColor)
+  }
+}
+
+@Composable
+fun SendReportRow() {
+  val context = LocalContext.current
+  val isDarkTheme = isSystemInDarkTheme()
+
+  Row(
+    modifier = Modifier
+      .fillMaxWidth()
+      .clickable {
+        (context as? Activity)?.start<DiagnosticReportActivity>()
+      },
+    verticalAlignment = Alignment.CenterVertically,
+    horizontalArrangement = Arrangement.Start
+  ) {
+    Image(
+      painter = painterResource(R.drawable.ic_feedback_orange_24dp),
+      contentDescription = stringResource(R.string.send_report),
+      modifier = Modifier
+        .padding(dimensionResource(R.dimen.activity_horizontal_margin))
+    )
+
+    Text(
+      text = stringResource(R.string.send_report),
+      color = if (isDarkTheme) Color.LightGray else Color.DarkGray,
+      fontSize = SendDiagnosticReportFontSize
```

Comment:
Instead of hardcoding the size of text use `style = MaterialTheme.typography.titleMedium` from the `KiwixTheme` or any better which fits in this.


Also, add the divider here at the last which will show the line at the bottom of "Send diagnostic report" to make the row completed.